.github/workflows/test.yml

```diff
@@ -271,6 +280,47 @@ jobs:
           command: check
           args: --workspace --all-targets
 
+  build_turborepo:
+    name: Build Turborepo
+    needs: [determine_jobs, rust_prepare]
+    if: needs.determine_jobs.outputs.go == 'true' || needs.determine_jobs.outputs.tests == 'true' || needs.determine_jobs.outputs.node == 'true'
```

Comment:
Why does this need to run when `node` has changed? If this depends on some JS files you can add these files to the `go` diff and remove that